ansible/roles/test/tasks/vlan_configure.yml

```diff
@@ -0,0 +1,89 @@
+- fail: msg="Please set ptf_host variable"
+  when: ptf_host is not defined
+
+- fail: msg="Invalid testbed_type value '{{testbed_type}}'"
+  when: testbed_type not in [ 't0' ]
+
+- debug: var=minigraph_portchannels
+
+- debug: var=minigraph_port_indices
+
+- name: Generate VLAN ports information
+  template: src=roles/test/templates/vlan_info.j2
+            dest=/tmp/vlan_info.yml
+  connection: local
+
+- name: Load VLAN ports info from file
+  include_vars: '/tmp/vlan_info.yml'
+
+- debug: var=vlan_ports_list
+- debug: var=vlan_intf_list
```

Comment:
can we move line 7-20 to vlantb.yml? I need those variables when I only run the vlan_test.yml